From 2edc0d682c6f352f15df0b3556f9ef122c0910df Mon Sep 17 00:00:00 2001
From: Andrey Borzenkov <arvidjaar@mail.ru>
Date: Thu, 4 Mar 2010 14:00:29 +0100
Subject: [PATCH] Fix boot with user suspend and no resume= kernel argument

Mandriva bug https://qa.mandriva.com/show_bug.cgi?id=57924. Check
if dracut got resume= argument before calling /usr/sbin/resume
---
 modules.d/95uswsusp/resume-uswsusp.sh | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/modules.d/95uswsusp/resume-uswsusp.sh b/modules.d/95uswsusp/resume-uswsusp.sh
index 9b18567..ce0e3d2 100755
--- a/modules.d/95uswsusp/resume-uswsusp.sh
+++ b/modules.d/95uswsusp/resume-uswsusp.sh
@@ -8,4 +8,6 @@ case "$splash" in
     ;;
 esac
 
-/usr/sbin/resume $a_splash "$resume"
+if [ -n "$resume" ]; then
+    /usr/sbin/resume $a_splash "$resume"
+fi
